setup.py

```diff
@@ -7,6 +7,7 @@
 
 requires = ['jmespath==0.7.1',
             'python-dateutil>=2.1,<3.0.0',
+            'rsa>=3.1.2,<=3.1.4',
```

Comment:
Quick feedback on this one. This line is currently copied from its counterpart in aws-cli. Later I did test the latest version of rsa 3.2 and it works. So I will lock to >=3.2,<3.3 in next PR.

But then it will become inconsistent with aws-cli. We may want to also upgrade rsa in aws-cli too. Coincidentally, there is a recent issue in our aws-cli repo suggesting an upgrade.